chosen/chosen.jquery.d.ts

```diff
@@ -1,30 +1,112 @@
-// Type definitions for Chosen.JQuery 1.4.2
+﻿// Type definitions for Chosen.JQuery 1.6.1
 // Project: http://harvesthq.github.com/chosen/
 // Definitions by: Boris Yankov <https://github.com/borisyankov/>
```

Comment:
Can you add your name to the authors as well? As this looks like a non-trivial change